airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -14,7 +14,11 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
-      validate_output_from_all_streams: yes
+      empty_streams:
+        - ads_insights
+        - ads_insights_age_and_gender
+        - ad_sets
+        - ads
       # FB serializes numeric fields as strings
       validate_schema: no
```

Comment:
you will not be able to reproduce original issue because this check is off here

Reply:
Ok, thanks :)